person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :age
+  attr_reader :id, :name
+
+  def initialize(id, age, name: 'Unknown', parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def name=(new_name)
+    @age = new_name
```

Comment:
* Kindly update this setter method for name, you should be setting `new_name` to @name instead of @age